org.winehq.Wine.DLLs.dxvk.yml

```diff
@@ -0,0 +1,96 @@
+id: org.winehq.Wine.DLLs.dxvk
+build-extension: true
+runtime: org.winehq.Wine
+runtime-version: stable-21.08
+sdk: org.freedesktop.Sdk//21.08
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.mingw-w64
+build-options:
+  prefix: /app/dlls/dxvk
+  prepend-path: /app/dlls/dxvk/bin
+  append-path: /usr/lib/sdk/mingw-w64/bin
+  #FIXME -fstack-clash-protection causes ICE
+  # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=90458
+  cflags: -fno-stack-clash-protection
+  cxxflags: -fno-stack-clash-protection
+  ldflags: ''
```

Comment:
What's the reason behind clearing ldflags? Does it fail with default set (`-Wl,-z,relro,-z,now -Wl,--as-needed`)?

Reply:
It does, mingw linker doesn't understand `-z`.